examples/ultralytics-yolo/deepsparse_utils.py

```diff
@@ -514,6 +511,22 @@ def yolo_onnx_has_postprocessing(model_path: str) -> bool:
     return all(num_dims > outputs_num_dims[0] for num_dims in outputs_num_dims[1:])
 
 
+def download_model_if_stub(path: str) -> str:
+    """
+    Utility method to download model if path is a SparseZoo stub
+    :param path: file path to YOLO ONNX model or SparseZoo stub of the model
```

Comment:
nit: add blank line between summary and params

Reply:
Fixed! Thanks for catching.